amazon-s3/5.8/modules/ROOT/pages/amazon-s3-connector-studio.adoc

```diff
@@ -115,7 +116,7 @@ It is not always possible to validate your credentials before the exact operatio
 This can vary based on the bucket name and other parameters.
 For example, the test connection can fail if your credentials have a restricted policy.
 
-[[basic-authentication]]
```

Comment:
(maybe you already checked) but i think we should keep these anchors in case other .adocs are referencing these sections. theres no downside to keeping them even if they are referenced right?

Reply:
this anchor is referenced in line 98 of the same .adoc too so we should keep it